packages/dashboard/README.md

```diff
@@ -26,6 +26,14 @@ export default App;
 
 Note, you must pass in a configuration object. Configuration objects can be created with the [Editor package](https://github.com/CDCgov/cdc-open-viz/tree/main/packages/editor).
 
+```
+sampleConfig = {

```

Comment:
This is not the right type of sample config. newViz: true is an internal flag. Recommend just link to a file in the /examples/ folder.

Reply:
This has been updated.